new_platforms/vscode-extension/README.md

```diff
@@ -163,11 +175,23 @@ data to Microsoft, you can set the `telemetry.enableTelemetry` setting to `false
 ## Known Issues
 
 * Building SGX enclaves is not currently supported.
-* On Linux, if you see errors in an Edge container build that involve `standard_init_linux.go:207`, you are likely cross-building a container.  Adding these packages solves the cross-build issue on Ubuntu 19.04:
+* On Linux, for Edge projects, you will need to enable cross-building.
```

Comment:
Perhaps simply reference the section above? You can link to the section as `[link](filename.md#sectionname)`.

Reply:
maybe i'll just take it out of the known issues.  at this point, it seems like more of a req than an issue.